### PR TITLE
Create .project-inventory.json

### PR DESCRIPTION
**Pull request by francoisvdv**

### CODE REVIEW

File: .project-inventory.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "classification": "cli-application",
+  "jiraKey": "ICOMP",
```
Comment (aadmathijssen) [inline .project-inventory.json]: I think `IDEVT` fits better.

File: .project-inventory.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "jiraComponent": "",
```
Comment (aadmathijssen) [inline .project-inventory.json]: I we use `IDEVT` as JIRA key, then this should be `php`.

File: .project-inventory.json (hunk 1 above)
Comment (francoisvdv) [reply .project-inventory.json]: Agreed, I changed it to IDEVT.

File: .project-inventory.json (hunk 2 above)
Comment (francoisvdv) [reply .project-inventory.json]: Updated the component to php.

**Comment by aadmathijssen** [comment]
The PHP project inventory has been deprecated.

**Pull request closed without merging**